src/data/real/sqrt.lean

```diff
@@ -336,6 +340,33 @@ open real
 
 variables {α : Type*}
 
+lemma exists_rat_sq_btwn_rat_aux (x y : ℝ) (h : x < y) (hy : 0 < y) :
+  ∃ q : ℚ, 0 ≤ q ∧ x < q^2 ∧ ↑q^2 < y :=
+begin
+  rw ←sqrt_lt_sqrt_iff_of_pos hy at h,
+  obtain ⟨q, hxq, hqy⟩ := exists_rat_btwn h,
+  have hq : (0 : ℝ) ≤ q := x.sqrt_nonneg.trans hxq.le,
```

Comment:
you can strengthen this to `0 < q`, I understand it's not really that relevant but why not